modules/gdnative/gdnative/variant.cpp

```diff
@@ -576,7 +576,9 @@ godot_variant GDAPI godot_variant_call(godot_variant *p_self, const godot_string
 	godot_variant raw_dest;
 	Variant *dest = (Variant *)&raw_dest;
 	Callable::CallError error;
-	memnew_placement_custom(dest, Variant, Variant(self->call(*method, args, p_argcount, error)));
+	Variant ret;
+	self->call(*method, args, p_argcount, ret, error);
+	memnew_placement_custom(dest, Variant, Variant(ret));
```

Comment:
I guess the `Variant(ret)` could be just `ret` no?

Reply:
for some reason I dont understand it did not compile without it, likely because its a placement constructor.